src/DataForm/DataMembership.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\DataForm;
+
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use Symfony\Component\Validator\Constraints as Assert;
+
+class DataMembership
+{
+    /**
+     * @Assert\NotBlank
```

Comment:
You should add all your constraints here !
The File constraints for example

Reply:
Added size and MIME type to file constraints in annotations.